presets/workspace/dependencies/requirements.txt

```diff
@@ -29,3 +29,4 @@ gputil
 setuptools
 psutil
 trl
+ruff
```

Comment:
I don't think we need to include ruff as part of the runtime dependency. Installing it during CI / local dev environment should be sufficient.